README.md

```diff
@@ -1 +1,40 @@
-# Hello-Microverse
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello World
+
+> In this project, I set up a "Hello world" repository. The goal here is to master all of the tools and best practices I learned about in previous steps. 
+
+
```

Comment:
It would be better if you add a screenshot to the readme file, please have a look at this [template](https://github.com/microverseinc/readme-template)